File: dev/front-end/vpl-fe-web-public/src/models/vehicle.ts
import { VehicleType } from "./model";

export class Vehicle {
    id!: number;
    name!: string;
    brandName!: string;
    brandLogo!: string;
    model!: string;
    productionYear!: number;
    modelYear!: number;
    type!: VehicleType;
}

File: dev/back-end/services/vpl-be-price-lists-service/Domain/IntegrationsModels/Vehicle.cs
namespace PriceListsService.Domain.IntegrationsModels
{
    public class Vehicle
    {
        public long Id { get; set; }
        public long BrandId { get; set; }
        public string BrandName { get; set; }
        public string BrandLogo { get; set; }
        public long ModelId { get; set; }
        public string Model { get; set; }
        public string Name { get; set; }
        public int ProductionYear { get; set; }
        public int ModelYear { get; set; }
        public int Type { get; set; }
    }
}
